### ISSUE TITLE
Can't seem to get it to start.

### ISSUE THREAD

**Issue by Phate6660** [LarsEKrueger/bite#3]
I have successfully compiled it, however it instantly closes or crashes upon startup. I've tried starting it up on it's own, as well as starting it up with various programs via `-e` but to no avail.

I have no idea what the issue might be.

Info requested via README:

- Rust version: `rustc 1.41.1 (f3e1a954d 2020-02-24)`
- Cargo version: `cargo 1.41.0 (626f0f40e 2019-12-03)`
- GCC version: `gcc (Gentoo 9.2.0-r2 p3) 9.2.0`
- Linux version: `Linux gentoo 5.5.7-gentoo-VALLEY #2 SMP PREEMPT Thu Mar 5 20:49:36 EST 2020 x86_64 Intel(R) Core(TM) i5-3470 CPU @ 3.20GHz GenuineIntel GNU/Linux`
- SHA1 (via `git rev-parse HEAD`): `2061165e5960c1005a9a1200913c8364f2976ced`

**Comment by LarsEKrueger** [comment]
Try to start it with BITE_LOG=trace biteThis will create a file bite_•••.log with the current date in the name.Inspect the file and maybe mail it to me. Censor any private Info before posting it.Also check view/mod.rs for the selected font. I guess if the exact font isn't installed, this could be the problem.I'll update the readme next week. I quite busy this week, so I appologize for longer response times in advance._______Dr. Lars KrügerSent from a mobile device.Am 07.03.20, 18:36 schrieb Ash <notifications@github.com>:

  I have successfully compiled it, however it instantly closes or crashes upon startup. I've tried starting it up on it's own, as well as starting it up with various programs via -e but to no avail. 
  I have no idea what the issue might be. 
  Info requested via README: 
   
   Rust version: rustc 1.41.1 (f3e1a954d 2020-02-24) 
   Cargo version: cargo 1.41.0 (626f0f40e 2019-12-03) 
   GCC version: gcc (Gentoo 9.2.0-r2 p3) 9.2.0 
   Linux version: Linux gentoo 5.5.7-gentoo-VALLEY #2 SMP PREEMPT Thu Mar 5 20:49:36 EST 2020 x86_64 Intel(R) Core(TM) i5-3470 CPU @ 3.20GHz GenuineIntel GNU/Linux 
   SHA1 (via git rev-parse HEAD): 2061165e5960c1005a9a1200913c8364f2976ced

**Comment by Phate6660** [comment]
> I quite busy this week, so I appologize for longer response times in advance.

It's no problem, I'm in no rush. Take your time, I definitely get what it's like to be really busy.

> I guess if the exact font isn't installed, this could be the problem.

Yep, that's what it was.

`[2020-03-09 07:26:25.379396 -04:00] T["main"] ERROR [src/main.rs:154] Can't init GUI: Can't find specified font`

I'll modify the font specified in `view/mod.rs` and see if it works.

Update: It works now. I don't have the Courier font, so I replaced it with Misc Fixed in `view/mod.rs`.

----

This is separate from the issue, but I thought I might say it while the issue is open. I'm really liking the terminal so far, though there are a few things I'm noticing. For example; I've noticed that colored output via escape sequences don't work. I've also noticed that some programs have no output in the terminal while some do. Like `neofetch` doesn't output anything at all (though it does get the green `O>>` above the prompt specifying that it ran properly), while `rsfetch` outputs everything almost perfectly. Also I was wondering, what's up with the colored blocks on the left side. I can see that they mean something as they change color, but I don't really get it. Also, the cursor seems to be under the prompt for some reason, and the prompt is sometimes double-drawn.

TL;DR: I really like it, and I can see myself using this (and hopefully contributing at some point), though I've noticed some possibly non-working things.

**Comment by LarsEKrueger** [comment]
I'll put the font config to the top of the roadmap. _______Dr. Lars KrügerSent from a mobile device.Am 09.03.20, 12:29 schrieb Ash <notifications@github.com>:

  Yep, that's what it was. 
  [2020-03-09 07:26:25.379396 -04:00] T["main"] ERROR [src/main.rs:154] Can't init GUI: Can't find specified font 
  I'll modify the font specified in view/mod.rs and see if it works.

**Comment by LarsEKrueger** [comment]
I fixed the bug with the missing error message.
I also improved the readme a bit to point readers to the explanation about the color seams and how to switch between output and error of programs. The heuristics to do that automatically will improve over time. I assume your `neofetch` issue is related to that. I put that as a reference program in the TODO list as well.
